src/algorithms/tracking/TrackSeeding.cc

```diff
@@ -92,6 +92,13 @@ std::vector<edm4eic::TrackParameters*> eicrecon::TrackSeeding::makeTrackParams(S
         // avoid float overflow for hits on a line
         continue;
       }
+      if ( std::hypot(X0,Y0) < std::numeric_limits<double>::epsilon() ||
+	!std::isfinite(std::hypot(X0,Y0)) ) {
```

Comment:
`!std::isfinite(std::hypot(X0,Y0))` - this condition is redundant given we already checked `X0` and `Y0` by this point.

Reply:
On a second thought, this is not redundant, there are cases that overflow (`std::hypot(std::numeric_limits<float>::max(), 1.0e35f)`)